<launch>
    <node pkg="joy" name="joy_node" type="joy_node" output="screen"/> 
    <node pkg="origarm_ros" name="joystick_node" type="joy_teleopt" output="screen"/>      
</launch>
